pegasus/sites.v3/code.org/public/css/maker-skinny-banner.scss

```diff
@@ -76,26 +77,7 @@
     }
   }
 
-  // There's a mysterious <p> tag being added around
-  // the right image on https://code.org/circuitplayground.
-  // This aligns it correctly to the bottom of the container.
```

Comment:
This is no longer needed since I updated the /circuitplayground page to a .haml file.